packages/webos/speech/VoiceControlDecorator.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {findDOMNode} from 'react-dom';
+import hoc from '@enact/core/hoc';
+
+/**
+ * Adds a callback for voice events
+ *
+ * @class VoiceControlDecorator
+ * @memberof webos/speech
+ * @hoc
+ */
+const VoiceControlDecorator = hoc({}, (config, Wrapped) => {
```

Comment:
No need to include an empty object for the default config. It can be omitted and just pass the callback as the first argument.

Reply:
I will fix it~